roles/confluent.ksql/files/ksql.yml

```diff
@@ -1,8 +1,75 @@
----
```

Comment:
lets keep this line

Reply:
Yep , that's what I am testing with a molecule run. As soon as that is done i will update and push.

Reply:
Updated